### PR TITLE
Css edit

### PR DESCRIPTION
**Pull request by ZeynebKoc**

### CODE REVIEW

File: src/Pages/HomePage.svelte [hunk 1]
```diff
@@ -265,74 +261,61 @@
...
         font-weight: normal;
         font-size: 16px;
         line-height: 26px;
-        margin-top: -2px;
     }
     .coffee-photos {
-        position: absolute;
-        margin-top: 55px;
-        margin-left: 13px;
+        margin-top: -17px;
```
Comment (erdemalpay) [inline src/Pages/HomePage.svelte]: Why negative margin?

File: src/Pages/HomePage.svelte [hunk 2]
```diff
@@ -265,74 +261,61 @@
...
     .danche-photo {
-        height: 177px;
+        height: 178px;
     }
 
     /* section-why-choose-us */
     .section-why-choose-us {
-        height: 728px;
-        margin-top: 116px;
+        width: 1280px;
```
Comment (erdemalpay) [inline src/Pages/HomePage.svelte:306]: If this is full width use 100%

File: src/Pages/HomePage.svelte [hunk 3]
```diff
@@ -399,19 +377,20 @@
...
+        margin-top: 83px;
+        margin-bottom: 11px;
     }
 
     /* section-how-it-works */
     .section-how-it-works {
-        height: 565px;
-        margin-top: 177px;
+        margin-top: 328px;
         margin-left: -65px;
```
Comment (erdemalpay) [inline src/Pages/HomePage.svelte]: Why negative margin?

File: src/Pages/HomePage.svelte [hunk 4]
```diff
@@ -422,26 +401,20 @@
...
     .how-it-works-timeline {
-        margin-top: 167px;
         width: 790px;
     }
     ul {
         display: flex;
         justify-content: space-between;
-        align-items: flex-start;
-        margin-left: -40px;
+        margin-left: -41px;
```
Comment (erdemalpay) [inline src/Pages/HomePage.svelte]: You know…

File: src/components/OrderSummaryPopUp.svelte [hunk 5]
```diff
@@ -61,7 +105,7 @@
         mix-blend-mode: normal;
         opacity: 0.5;
         z-index: 2;
-        height: 4136px;
+        height: 4696px;
```
Comment (erdemalpay) [inline src/components/OrderSummaryPopUp.svelte]: We should not have this height

File: src/components/Subscribe.svelte [hunk 6]
```diff
@@ -331,62 +336,52 @@
...
     }
     .how-it-works-content {
         width: 1045px;
         height: 453px;
         display: flex;
         flex-direction: column;
-        margin-top: -540px;
-        margin-left: -65px;
+        margin-top: 84px;
+        margin-left: -63px;
```
Comment (erdemalpay) [inline src/components/Subscribe.svelte]: ...

File: src/components/Subscribe.svelte [hunk 7]
```diff
@@ -400,7 +395,7 @@
         display: flex;
         justify-content: space-between;
         align-items: center;
-        margin-left: -40px;
+        margin-left: -41px;
```
Comment (erdemalpay) [inline src/components/Subscribe.svelte]: ...

File: src/Pages/HomePage.svelte (hunk 2 above)
Comment (ZeynebKoc) [reply src/Pages/HomePage.svelte:306]: The full width is 1440px. So I used 1280px.

File: src/Pages/HomePage.svelte (hunk 3 above)
Comment (ZeynebKoc) [reply src/Pages/HomePage.svelte:390]: In the main element I had set the page to align-items: center. But this section is not in the center. So I used negative margin to pull it to the left.

File: src/Pages/HomePage.svelte (hunk 4 above)
Comment (ZeynebKoc) [reply src/Pages/HomePage.svelte]: :) I removed.

File: src/components/Subscribe.svelte (hunk 7 above)
Comment (ZeynebKoc) [reply src/components/Subscribe.svelte]: I solved it by resetting the ul element in the global.css file

File: src/components/OrderSummaryPopUp.svelte (hunk 5 above)
Comment (ZeynebKoc) [reply src/components/OrderSummaryPopUp.svelte]: I fixed it

File: src/components/Subscribe.svelte (hunk 6 above)
Comment (ZeynebKoc) [reply src/components/Subscribe.svelte]: I removed negative margin.

File: src/Pages/HomePage.svelte (hunk 3 above)
Comment (ZeynebKoc) [reply src/Pages/HomePage.svelte]: I removed negative margin.

File: src/Pages/HomePage.svelte (hunk 1 above)
Comment (ZeynebKoc) [reply src/Pages/HomePage.svelte]: I removed negative margin.

**Pull request merged**